Signal/src/ViewControllers/HomeView/ConversationSearchViewController.swift

```diff
@@ -253,7 +253,10 @@ class ConversationSearchViewController: UITableViewController {
                 // a snippet for conversations that reflects the latest
                 // contents.
                 if let messageSnippet = searchResult.snippet {
-                    overrideSnippet = NSAttributedString(string: messageSnippet)
+                    overrideSnippet = NSAttributedString(string: messageSnippet,
+                                                         attributes: [
+                                                            NSAttributedStringKey.foregroundColor: Theme.primaryColor
```

Comment:
`HomeViewCell` uses attributed strings for its "subtitle/snippet" label. See `attributedSnippetForThread:`.  I suppose we could use `NSString` and consistent styling for "override snippets", but it's certainly consistent this way.  Shrug?

Reply:
I'm not against using attributes string, as I understand this makes it easier to prepend the various icon fonts that we sometimes use in the snippet text. But I was thinking that all the pieces of the attributed strings would be the same color, so it seems like we could apply the color at the label level, rather than to each piece of the attributed string. 

If that's for some reason not straight forward, feel free to ignore, it's not a major sticking point.

Reply:
Check out `attributedSnippetForThread`.  The colors aren't always uniform.

https://github.com/signalapp/Signal-iOS/blob/e602cf55a0042b1c1d8ccd3c05ddad6f448c5066/Signal/src/ViewControllers/HomeView/HomeViewCell.m#L344